README.md

```diff
@@ -118,6 +118,9 @@ When creating an instance of SubtitleOctopus, you can set the following options:
   occurs if browser doesn't support web workers). (Optional)
 - `debug`: Whether performance info is printed in the console. (Default:
   `false`)
+- `prescaleTradeoff`: Scale down (`< 1.0`) the subtitles canvas to `softHeightLimit` to improve speed, or scale it up (`> 1.0`) to improve quality. (Default: `null` - no scaling)
+- `softHeightLimit`: The height to which the subtitles canvas will be scaled. (Default: `1080`)
```

Comment:
If `prescaleHeightLimit < 1` we scale down but not less than `softHeightLimit` unless we already below `softHeightLimit`.

If `prescaleHeightLimit > 1` we scale up to `softHeightLimit` unless we already above `softHeightLimit`.

We are always limited by `hardHeightLimit`, so it doesn't relate to prescaling.

Reply:
Maybe I'm misunderstanding your comment or maybe you got confused by GitHub showing more than just the line I commented on.

The suggestion is to rename `softHeightLimit` to `prescaleHeightLimit` since it only has an effect for prescaling. `hardHeightLimit` can keep its name (and is not related to prescaling).

Reply:
Sorry, I forgot about the second part of your post while replying (_was at $work_).

The name is nice, but I don't really want to change the original commits - that's why I probably won't be able to transfer features from Jellyfin fork at all.

At this rate, you might want to switch to:
```js
prescale: {
    factor: <number>,
    height: <number>
}
```

Reply:
I've renamed the options:
```
'prescaleTradeoff' -> 'prescaleFactor'
'softHeightLimit' -> 'prescaleHeight'
'hardHeightLimit' -> 'maxHeight'
```